scripts/ci/get_image_tag_from_branch.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
+IMAGE_VERSION=$1
+BUILD_NUMBER=$2
+GIT_BRANCH=$3
+branch=`echo $GIT_BRANCH| sed 's|/|.|g'`  #not sure if docker accept / in the version
```

Comment:
I don't understand the comment. What do you mean?

Reply:
it was [copied](https://github.com/IBM/ibm-block-csi-driver/pull/344/files#diff-9b66a01dbbc80099d12d8bf0611934fb2bd85f500001f7f5e282f2c691d13080)